docs/sources/reference/templating.md

```diff
@@ -36,6 +36,27 @@ interpolation the variable value might be **escaped** in order to conform to the
 For example, a variable used in a regex expression in an InfluxDB or Prometheus query will be regex escaped. Read the data source specific
 documentation article for details on value escaping during interpolation.
 
+### Advanced Formatting Options
+
+In Grafana 5.0, a new syntax for variables was introduced to control the formatting of the interpolation.
+
+Syntax: `${var_name:option}`
+
+Filter Option | Example | Raw | Interpolated | Description
+------------ | ------------- | ------------- | -------------  | -------------
+`glob` | ${servers:glob} |  `'test1', 'test2'` | `{test1,test2}` | (Default) Formats multi-value variable into a glob (for Graphite queries)
```

Comment:
Maybe I'm confused here. Isn't `:regex` the default for Graphite queries? Or maybe only when using tags? Let's try and keep this documentation general and specify in graphite documentation that glob is default when not using tags and regex is the default when using tags.